stdlib/public/SDK/Windows/WinSDK.swift

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
Nitpick: It's 2018. (Specifically, the right copyright line for new files is "Copyright (c) 2018 Apple…". We're not good at remembering to do this.)

Reply:
Haha, you have caught me red-handed 😨.  I never write the copyright notices, I always steal them from other files.